testing/botan/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Alexander F Rødseth <xyproto@archlinux.org>
+# Contributor: Angel Velasquez <angvp@archlinux.org>
+# Contributor: Douglas Soares de Andrade <douglas@archlinux.org>
+# Contributor: d'Ronin <daronin@2600.com>
+# Contributor: Hexchain Tong <richard0053@gmail.com>
+# Contributor: Steve Holweg <skytep@gmail.com>
+
+pkgname=botan
+pkgver=2.3.0
+pkgrel=1
+pkgdesc="Crypto library written in C++"
+arch="all"
+url="https://botan.randombit.net/"
+license="BSD"
+depends="libgcc"
```

Comment:
you don't need specify shared libraries dependencies since auild will autodetect those. Please remove libgcc dependency and let abuild handle it.